NimbleSurvey/Generated/Assets.swift

```diff
@@ -0,0 +1,194 @@
+// swiftlint:disable all
```

Comment:
Please ignore this file

Reply:
At this point, if we ignore this file, app won't compile since I generated it from the command line but forgot to integrate into Xcode's build phases 😅

Fixed in [#53c688c](https://github.com/Thieurom/ic-surveys-ios/pull/31/commits/53c688cfbdb777cfe6c02290ca72fa53b3052a3b) 🙏